ads/_prefetch.js

```diff
@@ -0,0 +1,43 @@
+/**
```

Comment:
Maybe a bad idea. I kinda wanted this to stand out since it needs to be updated a lot.